src/ng/compile.js

```diff
@@ -2455,6 +2469,24 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
+    function addEventBindingDirective(directives, eventName, isEventDirective) {
+      if (!isEventDirective) return;
+
+      directives.push({
+        compile: function(tElement, tAttr) {
```

Comment:
yeah I think we're going to close this, although I believe Igor was going to do it. Well don't delete the branch just in case minds are changed later on =)